pkg/kubelet/volumes.go

```diff
@@ -142,6 +125,22 @@ func (kl *Kubelet) mountExternalVolumes(pod *api.Pod) (kubecontainer.VolumeMap,
 		if builder == nil {
 			return nil, errUnsupportedVolumeType
 		}
+
```

Comment:
Is there any benefit to knowing you have a builder (for mounting) before an attempt to attach is made?  Error handling would prevent an attachment for a non-existent builder.